ios/FluentUI/Table View/TableViewTokens.swift

```diff
@@ -149,3 +155,16 @@ open class TableViewCellTokens: ControlTokens {
     case small
     case medium
 }
+
+open class ActionsCellTokens: TableViewCellTokens {
+    /// The regular text color in an ActionsCell
+    open var regularTextColor: DynamicColor { globalTokens.brandColors[.primary] }
```

Comment:
does regular text color make it sound that it isn't branding color? or is this actual designer curated name?

Reply:
but in line 145 you defined as "mainBrandColor"